client/keys/delete.go

```diff
@@ -31,11 +39,18 @@ func runDeleteCmd(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	_, err = kb.Get(name)
+	info, err := kb.Get(name)
 	if err != nil {
 		return err
 	}
 
+	if info.GetType() == keys.TypeLedger || info.GetType() == keys.TypeOffline {
+		if err := kb.Delete(name, "yes"); err != nil {
```

Comment:
I still think we should prompt the user (just provide a different explanatory sentence) to confirm key deletion.

Reply:
Please review